.circleci/config.yml

```diff
@@ -45,12 +46,12 @@ jobs:
 
     steps:
       - restore_cache:
-          key: source-v6
+          key: source-v7
```

Comment:
What's this incrementing about? Do we have to create a new cache key every time there's a config.yml change or something?

Reply:
No, but this is the only way to clear the cache now. I'm not sure how often we need to clear the cache. I figured it wouldn't hurt.